src/utils/schedule.ts

```diff
@@ -0,0 +1,12 @@
+import { LocationEvents } from "types/venues";
+
+export const sortScheduleRoomsAlphabetically = (rooms: LocationEvents[]) => {
```

Comment:
Instead of a standalone function like this, this standalone function should only be the inner function passed to the .sort. Wrapping the rest in here like this makes it harder to use.

Reply:
Also.. would have to check docs/etc, but it looks like @malaikaaiyar ’s implementation may be better/more robust than this?


https://github.com/sparkletown/sparkle/pull/1740/files